processor-tests/src/test/java/org/infinispan/protostream/processor/tests/ProtoSchemaTest.java

```diff
@@ -169,6 +171,8 @@ public void testGeneratedInitializer() throws Exception {
       ProtobufUtil.toWrappedByteArray(ctx, new Note());
 
       assertTrue(ctx.canMarshall(SimpleClass.class));
+      assertTrue(ctx.canMarshall(OuterRecord.class));
+      assertTrue(ctx.canMarshall("firstTestPackage.OuterRecord.InnerEnum"));
```

Comment:
Just for completeness can we add `assertFalse(ctx.canMarshall("firstTestPackage.InnerEnum"));` just as well? This should then fail without the change right?